docs_manual/source/api_samplesheets.rst

```diff
@@ -23,6 +35,8 @@ API Views
 
 .. autoclass:: ProjectIrodsFileListAPIView
 
+.. autoclass:: RemoteSheetGetAPIView
```

Comment:
This is a legacy hack view which will be removed. It's been kept out of the documentation for a reason, to discourage anyone from using it :)